### PR TITLE
Completed step 1 and 2

### PR DESCRIPTION
**Pull request by NoSegfault**
Followed the instructions and completed step 1 and 2: add password to user and datastore. In detail, I've modified User class and anywhere the User constructor is called by adding passwords.

### CODE REVIEW

File: src/main/java/codeu/controller/LoginServlet.java [hunk 1]
```diff
@@ -74,7 +74,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     if (!userStore.isUserRegistered(username)) {
-      User user = new User(UUID.randomUUID(), username, Instant.now());
+      // TODO: add password
```
Comment (draspawn) [inline src/main/java/codeu/controller/LoginServlet.java:77]: would recommend the team complete review of code and merge these changes as they have impact on other assignments.

**Review by AndresGutierrez01 — APPROVED** [review]
Everything looks good to me!

**Review by draspawn — APPROVED** [review]
Meets the specs and looks good!

**Review by heatherbolze — APPROVED** [review]
Looks great!

**Review by gracefank — APPROVED** [review]
Awesome job! :)

**Pull request merged**